DTIPrep/DTI/DTI.pm

```diff
@@ -162,14 +162,14 @@ RETURNS:
 sub getAnatFile {
     my ($nativedir, $t1_scan_type)  = @_;
 
-    # Fetch files in native directory that matched t1_scan_type
-    my $anat_list   = DTI::getFilesList($nativedir, $t1_scan_type);
+    # Fetch files in native directory that matched t1_scan_type and MINC file type
+    my $anat_list = DTI::getFilesList($nativedir, "\_$t1_scan_type\_.*mnc\$");
```

Comment:
Shouldn't the pattern passed as argument be

`"\_$t1_scan_type\_*.mnc\$"`

(watch the position of the `*`)?

Reply:
this is a regex and if I understand correctly `.*` means that it matches any characters 0 or more times between the `$t1_scan_type` and `mnc` file extension.

Reply:
Actually, I think we want to check that the file name ends with this pattern:

1. An underscore, followed by
2. The scan type followed by
3. An underscore, followed by
4. Any non-zero length sequence of characters (except underscore, as these are used as delimiters in a MINC file name) and finally
5. The string `.mnc`

This is written as `"\_$t1_scan_type\_[^_]+\.mnc\$"`.

Makes sense?